rest_api/sawtooth_rest_api/routes.py

```diff
@@ -167,7 +167,7 @@ def _try_validator_request(self, message_type, content):
         Sends a protobuf message to the validator
         Handles a possible timeout if validator is unresponsive
         """
-        timeout = 5
+        timeout = 20
```

Comment:
Since we're responding to http requests, looking around at what browsers will time out, 300 seconds is the minimum (Firefox), everything else will wait longer (for example, https://support.microsoft.com/en-us/help/181050/internet-explorer-error-connection-timed-out-when-server-does-not-respond).  So we don't have to be stingy here.